Dockerfile

```diff
@@ -9,7 +9,8 @@ ENV DOCKER_DD_AGENT=yes \
     PYTHONPATH=/opt/datadog-agent/agent \
     DD_CONF_LOG_TO_SYSLOG=no \
     NON_LOCAL_TRAFFIC=yes \
-    DD_SUPERVISOR_DELETE_USER=yes
+    DD_SUPERVISOR_DELETE_USER=yes \
+    DD_CONF_PROCFS_PATH="/host/proc"
```

Comment:
Does this entry take the place of having to install a custom `procfs_path` in datadog.conf ?